.bazelrc

```diff
@@ -70,9 +70,26 @@ build --incompatible_strict_action_env
 
 test --build_tests_only
 
-# TODO: setup a new cache for OpenROAD
-# Read only remote cache, updated by OpenROAD team members and CI.
-#build --remote_cache=https://storage.googleapis.com/megaboom-bazel-artifacts
-#build --remote_cache_compression=true
+# --- Performance Optimizations ---
+# Use a high number of jobs and let the scheduler manage concurrency
+build --jobs=200
```

Comment:
Unnecessary but not harmful afaik

Reply:
If it works 👍 We will find out

Reply:
The bzl docs say that local cpu resources will naturally throttle this.  I expect it would only matter if we had remote execution and > 200 cpus.